phpBB/phpbb/group/helper.php

```diff
@@ -15,17 +15,58 @@
 
 class helper
 {
-	/** @var  \phpbb\language\language */
+	/** @var \phpbb\auth\auth */
+	protected $auth;
+
+	/** @var \phpbb\cache\service */
+	protected $cache;
+
+	/** @var \phpbb\config\config */
+	protected $config;
+
+	/** @var \phpbb\language\language */
 	protected $language;
 
+	/** @var \phpbb\event\dispatcher_interface */
+	protected $phpbb_dispatcher;
+
+	/** @var \phpbb\path_helper */
+	protected $phpbb_path_helper;
+
+	/** @var \phpbb\user */
+	protected $user;
+
+	/** @var string phpBB root path */
+	protected $phpbb_root_path;
+
+	/** @var string PHP file extension */
+	protected $php_ext;
+
 	/**
 	 * Constructor
 	 *
-	 * @param \phpbb\language\language $language	Language object
+	 * @param \phpbb\auth\auth $auth Authentication object
+	 * @param \phpbb\cache\service $cache Cache object
```

Comment:
Are you sure it's the cache service and not the driver_interface?

Reply:
I checked and you're in fact using the cache service.

Reply:
Yeah I have to `obtain_ranks()` which is within the `cache\service`.